native/cuda/main.h

```diff
@@ -68,7 +65,6 @@ int cv_gpu_cascade_get_min_neighbors(cv::Ptr<cv::cuda::CascadeClassifier>*);
 Size2i cv_gpu_cascade_get_max_object_size(cv::Ptr<cv::cuda::CascadeClassifier>*);
 Size2i cv_gpu_cascade_get_min_object_size(cv::Ptr<cv::cuda::CascadeClassifier>*);
```

Comment:
Mmm, since we are renaming everything to cuda, what about changing these to `cv_cuda_*`?